### PR TITLE
validate-query make it possible to opt-out

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

File: cluster/manifests/skipper/deployment.yaml [hunk 1]
```diff
@@ -94,6 +94,9 @@ spec:
         args:
           - "run.sh"
           - "skipper"
+{{ if eq .ConfigItems.skipper_validate_query "false" }}
+          - "-validate-query=false"
```
Comment (AlexanderYastrebov) [inline cluster/manifests/skipper/deployment.yaml]: Could be just `- "-validate-query={{ .ConfigItems.skipper_validate_query }}"` I guess

Comment (szuecs) [reply cluster/manifests/skipper/deployment.yaml]: yes

Comment (szuecs) [reply cluster/manifests/skipper/deployment.yaml]: fixed

**Comment by mikkeloscar** [comment]
:+1:

**Comment by gargravarr** [comment]
:+1:

**Pull request merged**